__tests__/app.test.js

```diff
@@ -228,3 +228,24 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("returns 200 and a list of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        expect(users).toBeInstanceOf(Array);
```

Comment:
line 238 check is redundant because using a forEach on line 240 is only possible if users is an array